src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/events/ActivitiesAndAppointmentsService.kt

```diff
@@ -36,7 +38,7 @@ class ActivitiesAndAppointmentsService(
       startTime = appointment.startTime,
       endTime = appointment.endTime,
       internalLocationId = nomisMappingService.getNomisLocationId(appointment.prisonLocationId)!!,
-      comments = appointment.comments,
+      comments = if (toggles.isMasterPublicAndPrivateNotes()) appointment.notesForPrisoners else appointment.comments,
       appointmentType = supportedAppointmentTypes.typeOf(appointment.bookingType()),
```

Comment:
Just thinking through this, particularly around the time the toggle is set.  An existing booking might have comments, we switch the toggle, and the next time its updated we no longer created those comments on the appointment, instead we populate it with (potentially empty) notesForPrisoners. Might be right .. but it feels like there might be a middle bit of if (notesForPrisoners is empty and comments is not empty) still use the comments? Something like that..